app/controllers/application_controller.rb

```diff
@@ -3,6 +3,7 @@ class ApplicationController < ActionController::Base
 
   include DeviseTokenAuth::Concerns::SetUserByToken
 
+  before_action :authenticate_user!, unless: :devise_controller?
```

Comment:
We usually use an `api_controller` that inherits from `application_controller` where we put the authentication.